src/parser/de.rs

```diff
@@ -0,0 +1,95 @@
+use std::collections::HashMap;
+
+use serde_json::Value;
+type PosValHolder = HashMap<String, Vec<(String, String)>>;
+
+pub fn de_kebab(qry: &str) -> String {
```

Comment:
can we use something like https://crates.io/crates/heck to do the case conversion?

Reply:
@meskill this will just increase compile time to add dependencies, we just need to convert it to lower camel case, and according to original issue, `foo-bar` should be converted to `fooBar`, but actually `-` is not recognized as separator, using heck we can only convert something like `foo_bar` to `fooBar` and `de_kebab` function also remove unwanted blank spaces to avoid further iterations

Reply:
even simple things as conversion could have range of gotchas that you need to consider when you're going to implement this. And such libraries usually have automated tests and behaviour verified by users. 
Anyway, we already using another crate `convert_case` in the repo if you concern about the build time